src/components/Icon/index.tsx

```diff
@@ -55,3 +55,49 @@ export const TwitterIcon = createIcon(
7eFN1BG+MtgZLmQ16r1DTsPpBHAcTz5Cl8ov1x8gza0pOz08uVTs27+faDAsTdmhfXw0soI+XWe8/u33Ll/x+mibzrai5HZFgv6uY6xkQdw1z9HZcdM0R5HFO5JYjnoFMCBFmeKJbs5ll6D0G8RNB5xhPxJVsgPQXcAFUWu2t33+i17+7rAAK6SvLmFk/TzYDkCHQWZkedifNyi9Rv9363/hAFh8RdGyvZPW34djDMW8G9V6ZpDAnfVKSyk7zFNWgnX5OtYSHdauanGywAj8QgbokLDHnPD9BYLEsrydpX217OIHiiC5BUMSeYiulq1BzJhPSle5tL9ld8arK82TeeUmQjGtIchph5tR2nldQLYofubirIjSR764+4iqJZfa7O/ng3RsabRQYX8OhAUoPzvVLrSzyRG6Mg6elVRfoz2wF4NQE2vD3jF4eXHYrMEHBiVa7gnVxo/rn8dR2oB1/8rF/LZZH6uxFQC67iQP2dl9C8WzRm9ZNmyaVzI8xt5g3bgyODJ25VhRJtd/4qs8G8pxohArnLjZ7HvP5TtURk+R+Wr2uskwMH0ouP6H3SEfLqBtr7AhUTf3+p4crXjyV85Qv5OOSO4/uNNPA/lem58hJOaLRq5V4ZU7hib2I+ScXmCgcofi/CBIUlbGsPaymsjupcF9N6motIx6hqidyvgaqVs6cu6m62gvqr/lPkngmFX0HeYHr2U+m5svwPCdOPFDa0RwaHBLq6oXW3CS8ujypvPrG11hS3gpDc8aY3y6PWOkE9yV5667c79My2bEXtZprsv6wgZNQI6rWqv40q4ch9VbyqtJHhHAQ6Emrd0U8ctfNWUJqJVekpZzhgCTqiAI02cDUgjdRIvHfdxJEszj5/e2KYzJvqN+FaGiPdWbEgOp4MxG/fYa7zNSEHwVzZUlpU0bZuwyI3cOjZBsY3LWa+PMAL+pZpKTCs3rgfgB/Q2xSuXfqRTTybb4wCb+9iKOi7cddrTgYDzWFb4ny9nUq4jutVhOBs4nlwxTqCzLuv537ZpS8cBDmNszpxlm3d5ha8gbiklIJSPRtqxPwaAow3pO1lIT6SYssAaxWEVtB7wKkJKYbi7tt5g2RoHfZ1efA9YRK0llNNrCsiVA7BNJ1Mz12OUBq+uZjYYbZ0f6ESmp0HHSn54kF3MVtIbrRwR6rUR8msWb1NCt9a2SxN4vl9NTdaTLeZ4RwGOJ+/kOfnRVlOnzPCWdme9wiW8sQXqhgym48lnsH5lO0rrRMDB4zJdvGt6VsijuCuf7yDQaRhw7JkGdMIreCc9Y/UJlafZB+u6tGKBOaDTDRkzW2sURo053kYfZkN0tPWieoyxSDwE0MG0VkgPNbCukLbdWLf4rWKDhsFtxabXdD49JqAJ4I/oZHYBNRuToVsOHYBKCJ537R/p4Fm6gYX01rrPekJ/dAbg5J9zPD/gu/XvkMYbLaFZNadA7ojEbo4nX2q/4fQfclz/s/Pm2efe6VTAUYrs69sIgZ5cyMfarjvNWG1ia2gYcOzjcAbojWohFIuhNp+AdqgY2dQ8gkwvVMOLCa6+SIE8CghpDW6a6zHtFLGVtLCp9Zq49lQf02/bC1R9jTtMmNqGgMbT1ciguv5mfwN0BmkPFtCfU3v72fWlBv6I/rstfHpYk0L656EU3nF2cpf6BPL/mK2khrmtxhlwNnDh35Bx5QFgHmj2cTHdj5GT4/qfdITfwGL46BpTgtFFYOdVSfQ79WTsaMBRQi+blp2/tNsRhVvaNFLcABd4x/W/MnFSTNfrTdNxuAEjlfEgrSy6/uKPXPpTt+IbawOI8j+PrSCZdtrDJL7xPNaukKIA7rU2ZI+mtmNRf4geZQGdxS6iBUZgNwpouOAiFafzIbVG0Qqnj0h53cH77kvsVzTXUHtzpzVoIqPoMIuUa3Mrnq1nWUihSmiH8pvYxhpwkD5AOQWIwkBW+Pso19km5G/kVmfB0jc5C+UJ2Zz/gCmdQQK4jEy1OUKuhxdbBhT/DQJn5wOO1rRa1xEFBO/eyoWER9qIHhrZh+4cIR/mXv746aJv7sRKMd3I01e6B9HkSFSm59/BIHwPC+iVJt6sYVjg0nq7ikcB4zNcZ8eKJLTUrtL3aPsOYiFdUHSDTjuq+3dRL2ewVSTHNJ+QToOApGdfUHloIkXQmsZ4l/oDCekQJ7VnQ55oJX2m+cZIM6A5LKL3sUE6mwV0X/HZSvMy828F8iGdzwbpQ2yAtm/FszRGgLMBU1mIcwG1C3flvjAuibxoafTbwLVdO71jSwQrKsJK5QJdQGqEDSmMJoAT6xqXZRbmD4dzQjUDdBqxxhBwNlSnt04jp7522TTHe5vLPXmk4/mXFN2n0+gOKRrU88A9/5uOkL6OecKa0ZK5XMgbDP3Q+JRaSzN+ptdc7R0wDmfTloo0E5H4IX2VDVLAAvodi1SyMzgtYMQCA1b6AFgeYaHKeon4lXNYQF9UQYcX03ap4jlqJWrtEbQPkf0DKmPlwWxQBSwiYBQxJJi6grv5oyxUAbV3sICuYQGtZCvo+KKh6xkzQx3XcrzRY2Ffc5h9lgV0RjG9NYDkL0XZMfr6GwtUsOeVxf74CgvpPewX9KZx7Y9zabpyEtGJ075VnO68tgjkCGKG7v9e7AuszWAk832GJHfDtKsV80Gc3uocK04znVicn8ccfbOfex2Rv9vx5FVcFIa5K0+BW3BXTu63rdu/Y6sdAuo0y/ow5JnZK3cG+WZG+F/Kuv55mBbjrv+njJtHfpjHuKuoXpA+AfFFl2dceRZ38//DvSX9XZ4/uxXAOctbvB0XhQEu/DX1Po7bfxqAMqlxWdG/Gxfy0npl4DhSYKdZX6pf37JpcD3vEvm+rJs/XL1MqLrlH5BuwvHkI9wtgJxV6U49E668kHvyeMf13z27R+7c3b1sk/Lys95eGbjEJ8mfdf1gzvzCjPL78OLkqDTZSfqTv8q6b5tffl+9/f8HVY4EozWeUCMAAAAASUVORK5CYII="
```

Comment:
isn't it better to add the icon to assets/icon
and then (ie): 
`import GoodReserveLogo from 'assets/icon/goodreserve.png'`

`xlinkHref = GoodReserveLogo`

Same for the voltage one